_posts/2021-04-14-release-statefun-3.0.0.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "Stateful Functions 3.0.0 Release Announcement"
+subtitle: "The Apache Flink community is happy to announce the release of Stateful Functions (StateFun) 3.0.0."
+date: 2021-04-14T08:00:00.000Z
+categories: news
+authors:
+- igalshilman:
+  name: "Igal Shilman"
+  twitter: "IgalShilman"
+- tzulitai:
+  name: "Tzu-Li (Gordon) Tai"
+  twitter: "tzulitai"
+---
+
+The Apache Flink community is happy to announce the release of Stateful Functions (StateFun) 3.0.0!  This release focuses on bringing remote functions to the front and center of StateFun. It is now easier, more efficient, and more ergonomic to write applications that live in a separate process to the StateFun cluster.
```

Comment:
This needs a sentence or two to introduce statefun to new users coming across this release announcement

Reply:
@knaufk maybe you have some ideas? I need to think about it

Reply:
Something like: "Stateful Functions provides a cross-platform kit for building stateful serverless applications. This release now focuses on ..."? 

Idk

Reply:
I think that is, we need an introduction sentence...
Drafting one now, filling in in a bit.

Reply:
Applied @StephanEwen's proposed intro paragraph posted below.